tests/test_get_avatar.py

```diff
@@ -15,7 +15,7 @@
 def load_test_cases(type):
 	res = []
 	for website in data:
-		if website == 'behance':
+		if website in ['behance', 'opensuse']:
```

Comment:
Uh well, this is not really a "solution". I see opensuse does provide profile icons, and they seem fetch-able.
```
<img src="http://connect.opensuse.org/mod/profile/icondirect.php?lastcache=1513970837&amp;joindate=1513619674&amp;guid=56910&amp;size=large" border="0"  alt="biscuitsnake" title="biscuitsnake"  />
```
from https://connect.opensuse.org/pg/profile/biscuitsnake .
This is different from existing ones so you're going to have to code it in.
Or an issue can be made for this to be resolved later.(Whatever the mentors prefer I guess)

Reply:
this was me forgetting to remove opensuse from the list after testing, sorry
i'm not sure how to code in what's required so I might leave it as it is